installer/default_platform.conf

```diff
@@ -435,13 +435,20 @@ bootloader_menu_config()
         mv $onie_initrd_tmp/tmp/onie-support*.tar.bz2 $demo_mnt/$image_dir/
 
         if [ "$firmware" = "uefi" ] ; then
-            secure_boot_state=$(mokutil --sb-state)
+            reg_sb_guid=$(efivar -l | grep "SecureBoot")
+            secure_boot_state=""
+            if echo "$reg_sb_guid" | grep -q "SecureBoot";then
+                secure_boot_state=$(efivar -d --name $reg_sb_guid)
+            else
+                echo "Current UEFI do not support Secure Boot feature"
+                secure_boot_state="0"
+            fi
             echo secure_boot_state=$secure_boot_state
-            if [ "$secure_boot_state" = "SecureBoot enabled" ]; then
-                echo "UEFI Secure Boot is enabled"
+            if [ "$secure_boot_state" -eq 1 ]; then
```

Comment:
not sure what this 1 means here. can we use macro?

Reply:
done